msg/PositionsStamped.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
+float32[] distances
+geometry_msgs/Vector3[] positions
```

Comment:
We'll want to use a geometry_msgs/Point for position, Vector3 is meant for a 3D vector (i.e. direction). They're basically the exact same thing, this is just a convention so it's easier to tell what the data represents.